cloudoperators/juno
# chore(utils): move useEndlessScrollList to juno-ui-components, depreacate juno-utils package

# Summary

This pull request deprecates the `juno-utils` package, transfers the most essential hook to `@cloudoperators/juno-ui-components`, and updates its usage within our applications (`greenhouse`, `supernova`, `example`, and `doop`).

# Changes Made

- Deprecated the `juno-utils` package (but did not delete it). Cause it is better to create a new one as separated PR which going to be only focus on mock-server lib
- Transferred the critical hook from `juno-utils` to `@cloudoperators/juno-ui-components`.
- Updated all instances of the hook in the following applications:
  - `greenhouse`
  - `supernova`
  - `example`
  - `doop`

# Related Issues

- #446 

# Testing Instructions

1. Run `npm i` to ensure dependencies are up-to-date.
2. Start the application with `turbo dev --filter=supernova`.
3. Navigate to the "Alerts" section.
4. Attempt to scroll down within the "Alerts" section to verify functionality.

# Checklist
